src/components/views/voip/VideoFeed.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+Copyright 2019 The Matrix.org Foundation C.I.C.
```

Comment:
Copyright is transferred to the foundation:
```suggestion
Copyright 2015, 2016, 2019 The Matrix.org Foundation C.I.C.
```